lib/adapters/mongodb/converters.js

```diff
@@ -6,7 +6,8 @@ const Hoek = require('hoek')
 const mongoose = require('mongoose')
 const uuid = require('node-uuid')
 
-module.exports = function() {
+module.exports = function(options) {
+    const baseUri = options.baseUri;
```

Comment:
We need to chose a style with regards to semicolons. I don't mind either. But you can free yourself from semicolons as long as you stick to one rule: never start a line with ```[``` or ```(``` instead use ```;[``` and ```;(```. You can ignore semicolons everywhere else.